### PR TITLE
create seed file and add carrierwave gem

### PR DESCRIPTION
**Pull request by kian1213**
Hi, please check my pull request. Thanks

### CODE REVIEW

**Comment by codegracie** [comment]
Okay

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <title><%= t ".title".html_safe %></title>
-  <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track": true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track": true %>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/application.html.erb]: convention

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/application.html.erb]: convention

File: config/application.rb [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 require 'rails/all'
```
Comment (MuhammadTamzid) [inline config/application.rb]: double quote

File: config/application.rb [hunk 4]
```diff
@@ -2,6 +2,11 @@
...
 
+require 'carrierwave'
```
Comment (MuhammadTamzid) [inline config/application.rb]: double quote

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <title><%= t ".title".html_safe %></title>
-  <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track": true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track": true %>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': true %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: Double quote

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -2,8 +2,8 @@
...
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': true %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: Double quote

File: db/seeds.rb [hunk 7]
```diff
@@ -5,12 +5,139 @@
...
+  password: "admin123",
+  password_confirmation: "admin123",
+  avatar: seed_image("default.jpg"),
+  admin: true
+)
+puts "Done creating default user"
+
+Categories = [
+  {
+    :name => "Basic 500",
```
Comment (dieunb) [inline db/seeds.rb]: `name: "Basic 500",`

**Pull request merged**